src/components/DateInput/Month.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import dayjs, { Dayjs } from 'dayjs';
+import { chunk } from 'lodash';
+import Flex from '../Flex';
+import Day from './Day';
+import { noop } from '../../utils/helpers';
+export interface MonthProps {
+  year: number;
+  month: number;
+  daySelected?: Dayjs;
+  onDaySelect?: (date: Dayjs) => void;
+}
+
+const Abbr = styled.abbr`
+  font-size: ${({ theme }) => theme.fontSizes.small};
+  font-weight: ${({ theme }) => theme.fontWeights.bold};
+  text-decoration: none;
+`;
+
+const Body = styled.div`
```

Comment:
all that though `<Box>`. It's easier to reason about (especially during mass updates to the API) when all of our components have the same "base" component.

It's also easier to write them